Lesson7/Task2/Horse.java

```diff
@@ -0,0 +1,35 @@
+package Lesson7.Task2;
+
+public class Horse extends Animal{
+    private String name;
+    private int age;
+
+    public Horse(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
```

Comment:
зайве можна прибрати так як воно є в Animal, а в конструкторі можна викликати просто батьківський конструктор
```suggestion
    public Horse(String name, int age) {
        super(name, age);
    }
```

Reply:
Дякую за підказку, змінила в усіх інших класах